modules/ROOT/pages/dataweave-formats-xml.adoc

```diff
@@ -749,7 +746,7 @@ cXmlObj
 ----
 <?xml version='1.0' encoding='UTF-8'?>
 <!DOCTYPE cXML SYSTEM "http://xml.cXML.org/schemas/cXML/1.2.014/cXML.dtd">
-<cXML payloadID="9949494" xmlns:xml="http://www.w3.org/XML/1998/namespace" xml:lang="en-US" timestamp="2002-02-04T18:39:09-08:00">
```

Comment:
Need to check with Ana if this should be removed or not: `xmlns:xml="http://www.w3.org/XML/1998/namespace" `